classes/utils/Utils.py

```diff
@@ -49,23 +60,26 @@ def __gt__(self, other):
         else:
             return self.all > other.all
 
-    v1_12_0 = (1, 12, 0, 335)
-    v1_12_1 = (1, 12, 1, 338)
-    v1_12_2 = (1, 12, 2, 340)
-    v1_13_0 = (1, 13, 0, 393)
-    v1_13_1 = (1, 13, 1, 401)
-    v1_13_2 = (1, 13, 2, 404)
-    v1_14_0 = (1, 14, 0, 477)
-    v1_14_1 = (1, 14, 1, 480)
-    v1_14_2 = (1, 14, 2, 485)
-    v1_14_3 = (1, 14, 3, 490)
-    v1_14_4 = (1, 14, 4, 498)
-    v1_15_0 = (1, 15, 0, 573)
-    v1_15_1 = (1, 15, 1, 575)
-    v1_15_2 = (1, 15, 2, 578)
-    v1_16_0 = (1, 16, 0, 735)
-    v1_16_1 = (1, 16, 1, 736)
-    v1_16_2 = (1, 16, 2, 751)
+    v1_12_0 = (1, 12, 0, 335, '1.12')
+    v1_12_1 = (1, 12, 1, 338, '1.12.1')
+    v1_12_2 = (1, 12, 2, 340, '1.12.2')
+    v1_13_0 = (1, 13, 0, 393, '1.13')
+    v1_13_1 = (1, 13, 1, 401, '1.13.1')
+    v1_13_2 = (1, 13, 2, 404, '1.13.2')
+    v1_14_0 = (1, 14, 0, 477, '1.14')
+    v1_14_1 = (1, 14, 1, 480, '1.14.1')
+    v1_14_2 = (1, 14, 2, 485, None)
+    v1_14_3 = (1, 14, 3, 490, '1.14.3')
+    v1_14_4 = (1, 14, 4, 498, '1.14.4')
+    v1_15_0 = (1, 15, 0, 573, '1.15')
+    v1_15_1 = (1, 15, 1, 575, '1.15.1')
+    v1_15_2 = (1, 15, 2, 578, '1.15.2')
```

Comment:
Same as before

Reply:
I'll leave that list in tact as it doesn't cause the data to be loaded and it could be helpful for us in the future